### ISSUE TITLE
ERROR: parse error: task includes cannot be used with other directives: sudo

### ISSUE THREAD

**Issue by YvesChan** [ansible/ansible#3632]
Hi there.
I'm using Ansible 1.2 but found a strange "parse error" problem.
Here's my playbooks:

``` yaml

---

- name: install necessary dependencies
  hosts: all
  gather_facts: False
  sudo: yes

  tasks:
    - name: create deploy user
      user: name={{ USER }} shell=/bin/bash

    - name: add authorized_key to deploy user
      authorized_key: user={{ USER }}
                      key="{{ lookup('file', 'files/id_rsa.pub') }}"

    - name: install fundamental software
      apt: pkg={{ item }} update_cache=yes
      with_items:
      - git-core
      - g++
      - curl
      - python-software-properties
      - make

    - name: create nvm directory
      file: dest={{ PREFIX }}/nvm owner={{ USER }} group={{ USER }} mode=0755 state=directory

    - name: create rbenv directory
      file: dest={{ PREFIX }}/rbenv owner={{ USER }} group={{ USER }} mode=0755 recurse=yes state=directory

    - include: node.yml user={{ USER }}
      sudo: no

    - include: ruby.yml user={{ USER }}
      sudo: no
```

No matter where I put the other two yml files(node & ruby) in, the problem still exists. 
I have google it the whole day but can not figure it out... 
Any help, please?

**Comment by mpdehaan** [comment]
It's true that sudo is not a keyword on the include statement in Ansible 1.2 and later.

Since you didn't specify the version, I can't say, but just set it on the play, start a new play if needed, or set it on the tasks.

IIRC, this is resolved in 1.3 (devel) latest, let me know if not.

Thanks!

**Comment by YvesChan** [comment]
Thanks @mpdehaan 
I have read the basics of ansible in http://www.ansibleworks.com/docs/ . But I can't find which factors a task could have. Include isn't a task? I used to think it's one of the modules...
And sometimes I find the {{ vars }} will not be parsed in conditional keywords like "when: ". Then I change to $vars and it doesn't work, either. How strange it is.... Maybe the FAQ documents should be improved :)
